www/docs/customization/templates.md

```diff
@@ -93,7 +93,7 @@ You should be able to use all its fields on each item:
 - `.Gomips64` (since v2.4)
 - `.Goppc64` (since v2.4)
 - `.Goriscv64` (since v2.4)
-- `.Go386` (since v2.4)
```

Comment:
i think it should actually be Go386 to match the others, let me see where I fucked it up